src/screencast/screencast.c

```diff
@@ -472,21 +472,24 @@ int xdpw_screencast_init(struct xdpw_state *state) {
 	state->screencast.state = state;
 
 	int err;
-	err = xdpw_pwr_core_connect(state);
+	err = xdpw_pwr_context_create(state);
 	if (err) {
-		goto end;
+		goto fail_pipewire;
 	}
 
 	err = xdpw_wlr_screencopy_init(state);
 	if (err) {
-		goto end;
+		goto fail_screencopy;
 	}
 
 	return sd_bus_add_object_vtable(state->bus, &slot, object_path, interface_name,
 		screencast_vtable, state);
 
-end:
-	// TODO: clean up pipewire
+fail_screencopy:
 	xdpw_wlr_screencopy_finish(&state->screencast);
```

Comment:
Nit: can you indent this line like the one below it? Labels get no indentation but their contents match the surrounding indentation.

Reply:
Not sure what you mean, this is how it looks in my editor:
![screenshot_2021-05-23-125422](https://user-images.githubusercontent.com/6507947/119257552-04993400-bbc6-11eb-9287-0c13666a5785.png)

No indent for the label iself (as you say), one tab for the code itself (matches the surrounding).

Reply:
Sorry, the GitHub mobile app lied to me.